ios/Classes/AnnotationController.swift

```diff
@@ -53,23 +53,42 @@ class AnnotationController: ControllerDelegate {
     func handleCreateManager(methodCall: FlutterMethodCall, result: @escaping FlutterResult) {
         guard let arguments = methodCall.arguments as? [String: Any] else { return }
         guard let type = arguments["type"] as? String else { return }
+        let id = (arguments["id"] as? String) ?? String(UUID().uuidString.prefix(5))
```

Comment:
Yeah, that's was my initial plan, however, that parameter is not optional, so it would make for quite a awkward unwrapping of the id from arguments and either calling create annotation manager with or without this parameter as a separate invocation. Seemed like a lot, so I rolled with this.
This is what I'm talking about:
```swift
let circleManager: CircleAnnotationManager
if let id {
    circleManager = mapView.annotations.makeCircleAnnotationManager(id: id, layerPosition: belowLayerId.map(LayerPosition.below))
} else {
    circleManager = mapView.annotations.makeCircleAnnotationManager(layerPosition: belowLayerId.map(LayerPosition.below))
}

/// same for symbol, line and polygon managers

```